src/DbPostgreSQLAdapter/Table.cpp

```diff
@@ -0,0 +1,660 @@
+#include "stdafx.h"
+#include "Table.h"
+
+#include "Database.h"
+#include "Field.h"
+#include "FieldValue.h"
+#include "PrimaryKey.h"
+#include "PrimaryKeyValue.h"
+#include "TableRecord.h"
+
+#include <DbAdapterInterface/IRecordSet.h>
+#include <DbAdapterInterface/IRecord.h>
+
+namespace {
+	std::string dateTimeToISOString(const std::chrono::system_clock::time_point& dateTime)
+	{
+		if (dateTime != std::chrono::system_clock::time_point{})
```

Comment:
If it does not enter the if clause, doesn't mean is "null"?

Reply:
Technically speaking, yes. The thing is that we don't have a "proper" definition of what represents Null. Right now we are using an empty string for it.